lib/ansible/module_utils/openswitch.py

```diff
@@ -21,9 +21,10 @@
 import json
 
 try:
-    from runconfig import runconfig
-    from opsrest.settings import settings
-    from opsrest.manager import OvsdbConnectionManager
+    import ovs.poller
+    import ops.dc
+    from ops.settings import settings
+    from opsr
```

Comment:
Hi @Qathos!

Thank you for the review and @mirmali will remove those in v2.  I asked him to push v2 today so that we can get all green.

Thank you for your support!